cmd/vl3-nse/vl3_connect.go

```diff
@@ -445,7 +445,7 @@ func removeDuplicates(elements []string) []string {
 }
 
 // newVL3ConnectComposite creates a new VL3 composite
-func newVL3ConnectComposite(configuration *common.NSConfiguration, vL3NetCidr string, backend config.UniversalCNFBackend, remoteIpList []string, getNseName fnGetNseName, defaultCdPrefix, nseControlAddr, connDomain string) *vL3ConnectComposite {
+var newVL3ConnectComposite = func(configuration *common.NSConfiguration, vL3NetCidr string, backend config.UniversalCNFBackend, remoteIpList []string, getNseName fnGetNseName, defaultCdPrefix, nseControlAddr, connDomain string) *vL3ConnectComposite {
```

Comment:
Nice. My understanding of this is that you want to modify the behavior of this function in the test which I really like. I'm only suggesting that moving this at the beginning of the file closer to the structs may be more visible that this is a var.

Reply:
Thanks! Sure, will do.